recipes/cubicinterpolation/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(KEEP_RPATH)
```

For better compatilibity on Mac